kubectl/README.md

```diff
@@ -39,6 +39,12 @@ first run a command to fetch cluster credentials as follows.
 
 Then, `kubectl` will have the configuration needed to talk to your GKE cluster.
 
+If your GKE cluster is in a different project than Container Builder, also set:
```

Comment:
add this to the section listing CLOUDSDK_COMPUTE_ZONE and CLOUDSDK_CONTAINER_CLUSTER - it's all the same "class" of configuration.

Reply:
👍  Fixed.